Assignment6/CParkingLot.cpp

```diff
@@ -0,0 +1,95 @@
+#include "CParkingLot.h"
+
+
+CParkingLot::CParkingLot() {
+    std::cout << "How many places are in the parking lot?" << std::endl;
```

Comment:
Yes, I think we should add it! I wanted to suggest it later when we would have chosen a solution:)